src/lucky_record/where.cr

```diff
@@ -140,4 +140,37 @@ module LuckyRecord::Where
       "#{column} #{operator} (#{prepared_statement_placeholder})"
     end
   end
+
+  class Raw
+    @clause : String
+    getter :statement
+
+    def initialize(@statement : String, *args)
+      ensure_enough_bind_variables_for! *args
+      @clause = @statement
+      build_clause *args
```

Comment:
Instead of setting @clause and then calling build_clause, it would probably be simpler to do something like this:

```crystal
@clause = build_clause(statement, bind_vars)
```

That way you don't need to set the `@statement` variable or set up a getter for it. You can make it a local variable instead

Reply:
nice, that's much cleaner!